src/libraries/System.IO.Pipes/src/System/IO/Pipes/NamedPipeClientStream.Windows.cs

```diff
@@ -106,6 +102,9 @@ private bool TryConnect(int timeout, CancellationToken cancellationToken)
             State = PipeState.Connected;
             ValidateRemotePipeUser();
             return true;
+
+            static SafePipeHandle CreateFileW(string? path, ref Interop.Kernel32.SECURITY_ATTRIBUTES secAttrs, int pipeFlags, int access)
```

Comment:
because `Interop.Kernel32.CreateNamedPipeClient` calls `CreateFileW` (there is no `CreateNamedPipeClient` sys-call).

I can rename it to `CreateNamedPipeClient` if you want.

Reply:
Either this should be renamed to CreateNamedPipeClient or https://github.com/dotnet/runtime/blob/c2ec86b1c552ac8a1749f9f98e012f707e325660/src/libraries/Common/src/Interop/Windows/Kernel32/Interop.CreateNamedPipeClient.cs#L14 should be renamed to CreateFileW.

Reply:
(I'd prefer to rename the import and then also see if we can just use the CreateFileW signature already defined elsewhere rather than having another file with another overload.)

Reply:
I've renamed it to `CreateNamedPipeClient`.

I've tried to remove `Interop.CreateNamedPipeClient.cs` and reuse `Interop.CreateFile.cs`, but we need to get a `SafePipeHandle` which is defined in `System.IO.Pipes` and not available in `SPC` so `Interop.CreateFile.cs` could not use `SafePipeHandle`. In theory I could make the following method to return an `IntPtr`:

https://github.com/dotnet/runtime/blob/2a95018d52fed9c6907f930e6151744c739b7321/src/libraries/Common/src/Interop/Windows/Kernel32/Interop.CreateFile.cs#L17

and reuse it from `System.IO.Pipes` and create `SafePipeHandle` out of it but I don't think it's worth it.